main.cpp

```diff
@@ -25,6 +29,7 @@ int main(  // NOLINT(bugprone-exception-escape): Using a big try-catch block to
                 // std::span is available in C++20.
       "A simple C compiler."};
   // clang-format off
+  cmd_options.custom_help("[options] file");
```

Comment:
Actually, I checked gcc and clang, that's why I use the current helper description. 🤔 And because we only support compiling a single file, I didn't add ... at the end of `file`.

```
// clang
OVERVIEW: clang LLVM compiler

USAGE: clang [options] file...

// gcc
Usage: gcc [options] file...
Options:
```

Reply:
Oops! Then let's follow our lovely Clang/GCC :nerd_face: